src/PageLayout/Notification/Recurring/RecurringDeliveryRequest2.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import truck from '../../../icons/green_truck.svg';
+
+class RecurringDeliveryRequest2 extends Component {
+    render() {
+        return (
+            <div className="modal-wrapper">
+                
+                <img className="icon" src={truck} alt="icon" />
+                <div className="modal-left-align">
+                    <h1>Are you sure you want to claim this pickup?</h1>
+                    <h2 className="warning">Once a pickup is claimed it cannot be cancelled.</h2>
+                    <p><span className="to-from">From:</span> Local Point      <span className="right to-from">To:</span> Seattle Union Gospel Mission <br/>
+                        {/* TODO: Backend replace info with timestamp */}
```

Comment:
see comment at line 16 in `RecurringDeliveryRequest1` about using that small component for these 3 lines